src/pages/CardPage.js

```diff
@@ -0,0 +1,336 @@
+import React from "react";
+import DefaultInternalTemplate from "../templates/DefaultInternalTemplate";
+import {
+  Breadcrumbs,
+  BreadcrumbItem,
+  BreadcrumbLinkItem,
+  SummaryList,
+  Card,
+  CardContent,
+  CardFooter,
+  Section
+} from "@baltimorecounty/dotgov-components";
+
+const publicWorksSummaryItems = [
+  {
+    id: 1,
+    icon: "fas fa-map-marker-alt",
+    heading: "Main Office",
+    content:
+      "<p> 111 West Chesapeake Avenue<br /> Suite 307<br /> Towson, Maryland 21204</p>"
+  },
+
+  {
+    id: 2,
+    icon: "far fa-clock",
+    heading: "Hours:",
+    content: "<p>Monday through Friday<br />8 a.m. to 4:30 p.m.</p>"
+  },
+  {
+    id: 3,
+    icon: "far fa-envelope",
+    heading: "Email:",
+    content: "<p><a href=`#`>publicworks@​baltimorecountymd.gov</a></p>"
+  },
+  {
+    id: 4,
+    icon: "fas fa-mobile-alt",
+    heading: "Phone:",
+    content:
+      "<p><a href=`tel:410-887-6789`>410-887-6789</a></p><p><h3>Emergency Dispatch:</h3></p><p><a href=`tel:410-887-6789`>410-887-6789</a></p>"
+  },
+  {
+    id: 5,
+    icon: "fas fa-exclamation-circle",
+    heading: "BALTCOGO311",
+    content: " <p><a href=`#`>Report a problem</a> online.</p>"
+  },
+  {
+    id: 6,
+    icon: "fas fa-user",
+    heading: "Director:",
+    content: "<p>Steve Walsh, P.E.</p>"
+  }
+];
+
+const mainContent = () => (
+  <Section>
+    <h2>Four Cards - Image, Text and Button</h2>
+    <div className="row">
+      <div className="d-flex col-md-4">
+        <Card>
```

Comment:
Since it looks like many of these cards are repeatable. You could so something like this.

```jsx
const sampleCard1 = props => (
  <Card>
    <CardContent>
      <svg
        width="100%"
        height="150px"
        xmlns="http://www.w3.org/2000/svg"
        preserveAspectRatio="xMidYMid slice"
        focusable="false"
        role="img"
        aria-label="Placeholder: Thumbnail"
      >
        <rect width="100%" height="200px" fill="#C2C2C2"></rect>
      </svg>
      <h3>Centered With Image</h3>
      <p>.8 acres</p>
      <p>
        <a href="https://www.google.com/maps">
          300 Lennox Avenue Room 202 Towson, Maryland 21286
        </a>
      </p>
    </CardContent>
    <CardFooter>
      <button type="button" class="dg_button">
        Learn More
      </button>
    </CardFooter>
  </Card>
);
```

Then refernce that component instead of repeating it. You can do that for each type of example that you have. That should clean up this file a bit.

Reply:
Let me know if that makes sense, if not, I can help you out.

Reply:
Updated based on our conversations and committed. Let me know if I should adjust further.